.git-commit-template

```diff
@@ -2,4 +2,6 @@ component: Subject
 
 Explanation
 
-https://pagure.io/freeipa/issue/XXXX
+Fixes: https://pagure.io/freeipa/issue/XXXX
+or
+Related: https://pagure.io/freeipa/issue/XXXX
```

Comment:
Hello @flo-renaud 
IMO, we should explain when to use fixes and related keyword.